### ISSUE TITLE
import with glob

### ISSUE THREAD

**Issue by singpolyma** [bos/configurator#10]
Would be useful to be able to import a glob pattern, especcially for the very common:

```
import conf.d/*
```

**Comment by taladar** [comment]
I agree that support for this idiom would be very useful, particularly when configuration is installed from several sources (e.g. something like logrotate.d or nagios rnpe.d) or for several logical services (virtual hosts,...) served by a single daemon or cronjob.

I would add that support for something like conf.d/*.cfg would be useful (a glob not just at the end) so files can just be renamed to something like *.cfg.disabled and stay where they are.

**Comment by bos** [comment]
This sort of support should be quite easy to add. I'll happily review and take such a patch.
